tensorflow/core/grappler/optimizers/auto_mixed_precision.cc

```diff
@@ -1056,7 +1056,7 @@ class AutoMixedPrecisionImpl {
       case AutoMixedPrecisionMode::CUDA:
         return std::make_unique<AutoMixedPrecisionListsCuda>(cuda_version_,
                                                              cudnn_version_);
-      case AutoMixedPrecisionMode::MKL:
+      case AutoMixedPrecisionMode::BF16:
```

Comment:
I think "ONEDNN" would be a better enum name than BF16, for consistency with the existing enum name CUDA. But for the sake of getting this in before the branch cut, we can address this later.

Reply:
Thanks @reedwm. yes, I can address this later. It's an internal change.